app/webpacker/stylesheets/application.scss

```diff
@@ -4,6 +4,7 @@
 @import 'header';
 @import 'users';
 @import 'about';
+@import 'books';
```

Comment:
you are experienced in the writing custom styles
let's have a deep dive into bootstrap: use bootstrap and only bootstrap for templates creating for the next couple of pages(e.g. book lists)

Reply:
also, I disagree with styles usage in that way:
you are defining some styles for classes with general names like `value`, `info-item` or even for tags `span` and `nav`. and imported those styles to `application.scss`. it's quite a high risk to override some styles in one of such importing files

possible solutions:
1. use namespaces: define all styles for books with book's class requirement:
```
.book {
  info { ... }
  nav { ... }
  ...
}
```
that was more or less common for previous rails assets pipeline
2. don't import everything to the main file, but use `stylesheet_pack_tag` for respective pages

P.S. originally, your project description recommends you using bootstrap and customizing only a few css classes. that is not an easier way, most likely you will be working with bootstrap on real projects